pkg/kn/commands/service_get_test.go

```diff
@@ -28,7 +29,21 @@ import (
 	client_testing "k8s.io/client-go/testing"
 )
 
-func fakeServiceGet(args []string, response *v1alpha1.ServiceList) (action client_testing.Action, output []string, err error) {
+func fakeServiceGet(args []string, response interface{}, listOrsingle bool) (action client_testing.Action, output []string, err error) {
+	var (
+		resp runtime.Object
+		ok   bool
+	)
+	if listOrsingle {
```

Comment:
both branches are equal, why do you need this `listOrsingle` variable ? (btw, would call it in all camelCase)

Reply:
Yes they are same..
Its my fault i was thinking some other scenario to handle and wrote that.
I will change and update